### DISCUSSION TITLE
help request: apisix 2.13.x  discovery with kubernetes

### DISCUSSION THREAD

**Discussion by wolgod** [apache/apisix#7817] [Q&A] [answered]
### Description

apisix discovery with kubernetes It's a new feature in 2.13, I don't know how stable it is,but 2.15 version is the latest version, now  I can only use the 2.13.x version of apisix. but 2.15 changelog  has fixed a bug. I don’t know how much this bug affects. If I don’t upgrade, will there be a big problem? 

### Environment

- APISIX version (run `apisix version`): 2.13.2
- Operating system (run `uname -a`): centos7.6
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`): 1.9.19
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

**Comment by tzssangglass** [comment, marked as answer]
2.13.3 has been released, changelog: https://github.com/apache/apisix/blob/release/2.13/CHANGELOG.md#2133

include the bugfix that you want?

**Reply by wolgod** [reply]
thanks，i will try it
